### ISSUE TITLE
Alphabetization warning incorrectly reported

### ISSUE THREAD

**Issue by gscottolson** [rossPatton/stylint#234]
`cat foo.styl` →
```
a
    height: 100px
    width: 100px
```

`cat bar.styl` →
```
a
    span
        height: 100px
        width: 100px
```

`stylint foo.styl` → :+1: 

`stylint bar.styl` →
```
Warning: prefer alphabetical when sorting properties
File: ./bar.styl
Line: 3: height: 100px

Warning: prefer alphabetical when sorting properties
File: ./bar.styl
Line: 4: width: 100px

Stylint: 0 Errors.
Stylint: 2 Warnings.
```

Why does this happen for nested rules?

**Comment by rossPatton** [comment]
can i see your config

rules like `alphabetical` and `duplicates` need to have `indentPref` set correctly to work

**Comment by gscottolson** [comment]
```json
{
  "blocks": false,
  "brackets": "never",
  "colons": "always",
  "colors": "always",
  "commaSpace": "always",
  "commentSpace": "always",
  "cssLiteral": "never",
  "depthLimit": false,
  "duplicates": true,
  "efficient": "always",
  "extendPref": false,
  "globalDupe": false,
  "indentPref": false,
  "leadingZero": "never",
  "maxErrors": false,
  "maxWarnings": false,
  "mixed": false,
  "namingConvention": false,
  "namingConventionStrict": false,
  "none": "never",
  "noImportant": true,
  "parenSpace": false,
  "placeholders": "always",
  "prefixVarsWithDollar": "always",
  "quotePref": false,
  "semicolons": "never",
  "sortOrder": "alphabetical",
  "stackedProperties": "never",
  "trailingWhitespace": "never",
  "universal": false,
  "valid": true,
  "zeroUnits": "never",
  "zIndexNormalize": false
}
```

I pulled this off the [Options example](https://github.com/rossPatton/stylint#options). I’d be happy to update the README with these dependencies so it’s clearer for future adopters. Just let me know.

**Comment by rossPatton** [comment]
feel free - I rarely have time to work on this project and any help is great appreciated!
